NDTensors/ext/NDTensorsAMDGPUExt/set_types.jl

```diff
@@ -1,11 +1,12 @@
 # TypeParameterAccessors definitions
-using NDTensors.TypeParameterAccessors: TypeParameterAccessors, Position
+using NDTensors.TypeParameterAccessors:
+  TypeParameterAccessors, Position, default_type_parameters
 using NDTensors.GPUArraysCoreExtensions: storagemode
 using AMDGPU: AMDGPU, ROCArray
+using GPUArraysCore: AbstractGPUArray
 
 function TypeParameterAccessors.default_type_parameters(::Type{<:ROCArray})
-  return (Float64, 1, AMDGPU.Mem.HIPBuffer)
+  return (default_type_parameters(AbstractGPUArray)..., AMDGPU.Mem.HIPBuffer)
```

Comment:
```suggestion
  return (default_type_parameters(AbstractArray)..., AMDGPU.Mem.HIPBuffer)
```
and same thing elsewhere. Then we wouldn't have to load `GPUArraysCore` above.

Reply:
This makes sense, I updated it. Thanks!